test/eth.methods.js

```diff
@@ -8,7 +8,7 @@ describe('web3', function() {
         u.methodExists(web3.eth, 'getState');
         u.methodExists(web3.eth, 'getStorage');
         u.methodExists(web3.eth, 'getTransactionCount');
-        u.methodExists(web3.eth, 'getCode');
+        u.methodExists(web3.eth, 'getData');
```

Comment:
`getData` is not verbose enough for me. `getCode` implicitly told me, that I want to get it from some address. `getData` doesn't do that.